overlay/triplets/x64-osx-mixxx.cmake

```
@@ -0,0 +1,19 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
+set(VCPKG_CRT_LINKAGE dynamic)
+
+# Mixxx loads fdk-aac dynamically at runtime. This allows the user to replace 
+# the version of fdk-aac we ship which has the patent-encumbered HE-AAC 
+#removed with another build that supports HE-AAC.
+if(${PORT} MATCHES "fdk-aac")
+    set(VCPKG_LIBRARY_LINKAGE dynamic)
+else()
+    set(VCPKG_LIBRARY_LINKAGE static)
+endif()
+
+set(VCPKG_CMAKE_SYSTEM_NAME Darwin)
+set(VCPKG_OSX_ARCHITECTURES x86_64)
+
+set(VCPKG_OSX_DEPLOYMENT_TARGET 10.12)
+set(VCPKG_C_FLAGS -mmacosx-version-min=10.12)
+set(VCPKG_CXX_FLAGS -mmacosx-version-min=10.12)
```

```suggestion
set(MACOS_MINUMUM 10.12)
set(VCPKG_OSX_DEPLOYMENT_TARGET ${MACOS_MINIMUM})
set(VCPKG_C_FLAGS -mmacosx-version-min=${MACOS_MINIMUM})
set(VCPKG_CXX_FLAGS -mmacosx-version-min=${MACOS_MINIMUM})
```
to ensure consistency

---

Nevermind, this code does not work. I tried the same for Tenacity and the ffmpeg build failed:
`clang: error: invalid version number in '-mmacosx-version-min='`